packages/widgetbook/lib/src/settings_panel/settings_panel.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/src/constants/constants.dart';
+import 'package:widgetbook/src/constants/radii.dart';
+import 'package:widgetbook/src/knobs/knobs_panel.dart';
+import 'package:widgetbook/src/properties/property_panel.dart';
+
+class SettingsPanel<CustomTheme> extends StatelessWidget {
+  const SettingsPanel({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 2,
+      child: Column(children: [
+        SizedBox(
+          height: Constants.controlBarHeight,
+          child: TabBar(
+            indicatorColor: Theme.of(context).colorScheme.primary,
+            tabs: const [
+              Tab(
+                key: Key('knobsTab'),
+                icon: Icon(
+                  Icons.build,
+                ),
+              ),
+              Tab(
+                key: Key('propertiesTab'),
+                icon: Icon(
+                  Icons.layers,
+                ),
+              ),
+            ],
+          ),
+        ),
+        const SizedBox(
+          height: 16,
+        ),
+        Expanded(
+          child: Container(
+            decoration: BoxDecoration(
+              color: Theme.of(context).colorScheme.surface,
+              borderRadius: Radii.defaultRadius,
+            ),
+            child: Padding(
+              padding: const EdgeInsets.all(16),
+              child: TabBarView(
+                children: [
+                  const KnobsPanel(),
+                  PropertyPanel<CustomTheme>()
+                ],
+              ),
+            ),
```

Comment:
Due to this Padding the Design of the PropertyPanel is changed because of the `Divider`s

Before (Design): 

<img width="384" alt="image" src="https://user-images.githubusercontent.com/16862002/163113244-f421a862-8636-4098-8390-933a8b130c1e.png">

Now: 
<img width="417" alt="image" src="https://user-images.githubusercontent.com/16862002/163113305-6b2f27aa-f8a0-4711-a299-e8e4585e8740.png">